lib/main.js

```diff
@@ -41,39 +72,102 @@ export default {
         const filePath = textEditor.getPath();
         const fileText = textEditor.getText();
 
-        let ruleset = rulesets;
-        if (/^[a-z0-9]+\.xml$/gi.test(rulesets)) {
-          const rulesetPath = await helpers.findAsync(filePath, rulesets);
-          if (rulesetPath !== null) {
-            ruleset = rulesetPath;
+        if (fileText === '' || filePath === null) {
```

Comment:
It can also be `undefined`, this is one of the cases where it should just be `if (fileText === '' || !filePath)`.